src/scenes/AcceleratorRouter/Documents/PreviewView/util/markdown-table.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from 'src/types/documentProducer/VariableValue';
 
 export const hasMarkdownTableHeaderSeparatorRow = (input: string | undefined): boolean =>
+  /* eslint-disable-next-line no-useless-escape */
   !!/\|\s*\-\-\-+\s*\|/g.exec(input || '');
```

Comment:
I think for this one we can actually change `\-\-\-` to just `---`. In regex, the `-` character doesn't need to be escaped if it's not inside square brackets `[]`. I tested locally and the tests still pass

Reply:
Yeah, I considered that.  But since character classes could cause `-` to need an escape I opted against it and it doesn't hurt anything.